### PR TITLE
Fixed and enhanced the Navigators example

### PR DESCRIPTION
**Pull request by lovato**
It took me a few timeslots of my life to figure out how to make this work. First, I tried to break into 2 files. The app export made no sense to me. But after doing that, I discover a invalid token, the return() was not supposed to be there. So I fixed the sample.

### CODE REVIEW

**Comment by facebook-github-bot-0** [comment]
By analyzing the blame information on this pull request, we identified @hramos and @scarlac to be potential reviewers.

**Comment by facebook-github-bot-0** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-8** [comment]
@lovato updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9802/files/81347debbcb106b5dcf1c625e89df48f6297f5cb..624637b8fce78466c0e0a8f96b0a567d72a71d10)

**Comment by facebook-github-bot-9** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by facebook-github-bot-1** [comment]
@lovato updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9802/files/624637b8fce78466c0e0a8f96b0a567d72a71d10..70897a777ccf9bc7976f65665d8bd892bb862fd4)

**Comment by lovato** [comment]
CircleCI says:

Version '6.2.0' not found - try `nvm ls-remote` to browse available versions.
nvm install 6.2.0 && nvm alias default 6.2.0 returned exit code 3
Action failed: set node.js version to 6.2.0

What am I supposed to do?

File: docs/UsingNavigators.md [hunk 1]
```diff
@@ -100,46 +100,55 @@ navigator.push({
 navigator.pop();
 ```
 
-A more complete example that demonstrates the pushing and popping of routes could therefore look something like this:
+A more complete example that demonstrates the pushing and popping of routes. Edit your index*.js file to therefore look something like this:
```
Comment (mkonicek) [inline docs/UsingNavigators.md]: nit: Please remove the word "therefore" here.

**Comment by mkonicek** [comment]
Looks good, just one small comment. Thanks a lot for doing this!

**Comment by mkonicek** [comment]
The CircleCI failure is likely transient. I restarted the build.

**Comment by mkonicek** [comment]
I updated your PR title to mention the Navigator and also removed the template. Hope you don't mind!

**Comment by facebook-github-bot-1** [comment]
@lovato updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9802/files/70897a777ccf9bc7976f65665d8bd892bb862fd4..c7dee785791d160047f766cd6f9d6b4853acea37)

**Comment by mkonicek** [comment]
Thanks!

@facebook-github-bot shipit

**Comment by facebook-github-bot-2** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3841320) to review internal test results.

**Pull request closed without merging**